bin/make-consensus.py

```diff
@@ -77,8 +78,8 @@ def main():
               file=sys.stderr)
         sys.exit(0)
 
-    if args.maskNoCoverage and not args.bam:
-        print('If --maskNoCoverage is used, --bam must be too.',
+    if args.maskLowCoverage and not args.bam:
```

Comment:
Could (optionally) add a check that `args.maskLowCoverage` is not `< 0`.  The `if maskLowCoverage` elsewhere will be True if a value like -10 is passed in (which it probably never will be..... so optional).